waspc/data/Generator/templates/server/src/auth/providers/email/signup.ts

```diff
@@ -116,7 +116,9 @@ export function getSignupRoute({
             await createUser(
                 providerId,
                 newUserProviderData,
-                userFields,
+                // Using any here because we want to avoid TypeScript errors and
+                // rely on Prisma to validate the data.
+                userFields as any
```

Comment:
Added this `as any` since we had the same coerce in the username auth impl. Added the same explainer comment as there.